src/test/java/org/apache/commons/collections4/multimap/AbstractMultiValuedMapTest.java

```diff
@@ -60,6 +60,13 @@
     /** MultiValuedHashMap created by reset(). */
     protected MultiValuedMap<K, V> confirmed;
 
+    /**
+     * Flag to indicate the map makes no ordering guarantees for the iterator. If this is not used
+     * then the behaviour is assumed to be ordered and the output order of the iterator is matched by
+     * the toArray method.
+     */
+    protected static final int UNORDERED = 0x1;
```

Comment:
Remove this. The property to use when configuring the behaviour of the AbstractCollectionTest should be the flag defined in AbstractCollectionTest.